.circleci/config.yml

```diff
@@ -371,42 +380,54 @@ jobs:
 
   api-tests-prod-eu:
     docker:
-      - image: circleci/python:2.7
+      - image: python:2.7
     environment:
       GOOGLE_PROJECT_ID: open-targets-api-prod-eu
-    working_directory: ~/circulate
     steps:
       - checkout
       - run:
-          name: Run runscope tests
+          name: Install requests library
+          command: pip install requests
+      - run:
+          name: Trigger runscope tests on deployed branch
           command: |
-            source .circleci/runscope/run.sh
+            set -o allexport
+            source VERSION
+            python .circleci/runscope.py
 
   api-tests-prod-us:
     docker:
-      - image: circleci/python:2.7
+      - image: python:2.7
     environment:
       GOOGLE_PROJECT_ID: open-targets-api-prod-eu
-    working_directory: ~/circulate
     steps:
       - checkout
       - run:
-          name: Run runscope tests
+          name: Install requests library
+          command: pip install requests
+      - run:
+          name: Trigger runscope tests on deployed branch
           command: |
-            source .circleci/runscope/run.sh
+            set -o allexport
+            source VERSION
+            python .circleci/runscope.py
   
   api-tests-prod-jp:
     docker:
-      - image: circleci/python:2.7
+      - image: python:2.7
     environment:
       GOOGLE_PROJECT_ID: open-targets-api-prod-eu
```

Comment:
should this google project id be different rather than the same for japan, us and eu? @elipapa

Reply:
that would be a resounding yes! 👍 good catch